examples/redis-operator.md

```diff
@@ -0,0 +1,115 @@
+# Redis-pmem operator
+This readme describes a complete example to deploy a Redis cluster through the [redis-operator](https://github.com/spotahome/redis-operator) using QEMU-emulated persistent memory devices.
+
+## Prerequisites:
+* Docker-ce: version 18.06.1
+* [Docker registry](https://docs.docker.com/registry/deploying/).
+
+## PMEM-CSI installation
+The steps below describe how to install PMEM-CSI within a K8S cluster that contains one master and three worker nodes which use a specific version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced.
```

Comment:
At the end, change to:
for the nodes.
[delete ", so the following steps can be reproduced."]

Reply:
There are many reasons to choose an specific version of some SW in general, in this case we use an specific version so the steps can be reproduced, so I would like to keep this part of the sentence or rephrased if needed.

Reply:
Ok, I understand. Please rephrase one more time, then LGTM/OK to merge.  

The steps below describe how to install PMEM-CSI within a K8S cluster that contains one master and three worker nodes. We use a specific version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced.